tests/sync/test_rest_api.py

```
@@ -349,6 +349,113 @@ async def test_denial_end_to_end(self):
+        await seb_communicator.send_json_to(
+            {
+                "email": email,
+                "semi_sekret": semi_sekret,
+                "denial_id": denial_id,
+            }
+        )
+        # Consume entity extraction responses
+        try:
+            while True:
+                response = await seb_communicator.receive_from(timeout=30)
+        except:
```

Ruff (0.14.6)

Do not use bare `except`

(E722)